docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+db:
+  image: postgres
+  ports:
+    - "5432"
+web:
+  build: .
+  command: >
+    /bin/bash -c '
+    sleep 3 &&
+    python manage.py migrate --noinput &&
+    python manage.py runserver 0.0.0.0:8077'
+  volumes:
+    - .:/src
+  environment:
+    PORT: 8077
+    DATABASE_URL: postgres://postgres@db:5432/postgres
+  ports:
+    - "8077:8077"
+  links:
+    - db
+dredd:
+  image: ccxcon_web
+  command: /bin/bash -c 'sleep 4 && dredd'
```

Comment:
This is causing the containers to close once `dredd` has finished executing after running `docker-compose up`. Can you remove the command here and call it from `travis.yml` directory, or figure out some other way to keep the containers from stopping?